functions.yml

```diff
@@ -116,7 +116,25 @@ etl:
           doc: Minimum fraction of bases that must remap to lift over successfully
           type: double
           is_optional: true
-          
+    - name: normalize_variant
+      doc: Normalizes the variant like vt decompose -s
```

Comment:
Can you improve the documentation? It would be nice to have a description of the algorithm as well as details about what the different result fields look like if there is / is not an error. It's actually better to put documentation here than on the scala transformer class.

Reply:
Added more doc here. Kept the one in transformer as well because it has that replace_columns option which makes the doc a bit different.